src/rfquack_radio.h

```diff
@@ -123,28 +128,21 @@ void rfquack_update_sync_words() {
  *      - set the modem registers
  *      - set preamble length and sync words
  */
-void rfquack_radio_setup() {
-  rfquack_rf.setDebug(RFQUACK_DEBUG_RADIO);
-  rfquack_rf.setPrinter(&LogPrinter);
+void rfquack_radio_setup(RFQRadio *rfqRadio = rfquack_rf) {
```

Comment:
Do we really need to take the pointer from the user? Can't we instantiate the object in here? We know the pin assignment, right?